src/towncrier/test/test_format.py

```diff
@@ -327,6 +331,38 @@ def test_line_wrapping_disabled(self):
         output = render_fragments(
             template,
             None,
+            "",
+            fragments,
+            definitions,
+            ["-", "~"],
+            wrap=False,
+            versiondata={"name": "MyProject", "version": "1.0", "date": "never"},
+        )
+        self.assertEqual(output, expected_output)
+
+    def test_title_format_specified(self):
+        """
+        Title format replaces default top line rendering.
+        """
+        self.maxDiff = None
+
+        fragments = {}
+
+        definitions = OrderedDict()
+
+        expected_output = u"""A custom top line
+=================
+"""  # NOQA
+
+        template = pkg_resources.resource_string(
+            "towncrier", "templates/default.rst"
+        ).decode("utf8")
+
+        fragments = split_fragments(fragments, definitions)
```

Comment:
Maybe use this ... in this way e have a hint that we don't care about fragments or definitions in this test and you don't need to look and see where and how they are created and how they are checked at the end.

```suggestion
        fragments = split_fragments(fragments={}, definitions=OrderedDict())
```

But for this tests it would be interesting to also have at least one definition and one fragment to increase the coverage and make it easy to compare the output for the case in which you run without a custom topline.